src/components/token/Tokens.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import ReactLoading from 'react-loading';
+import { Link } from 'react-router-dom';
+import colors from '../../index.scss';
+import TokenRow from './TokenRow';
+import {
+    shouldRenderCustomTokens
+} from '../../feature';
+
+/**
+ * Displays custom tokens in a table with pagination buttons and a search bar.
+ */
+class Tokens extends React.Component {
```

Comment:
I have the same comment about the prop types here

Reply:
Added PropTypes in the new commit.